### ISSUE TITLE
Kinda Sorted list of method names.

### ISSUE THREAD

**Issue by demmings** [clenemt/docdash#114]
The method names of the class are sorted to a point, then it starts over again.  (unless this is intended behavior).

You can see in this example, it is sorted up until **whereCondition** and then starts in a sorted order from **dateToMs**

![image](https://user-images.githubusercontent.com/87726024/211214978-1ee13240-cd31-4330-ba7c-818b2a54e7a7.png)

**Comment by ar2rsawseen** [comment]
Yes, I think properties are also there in the mix before methods.
But at least properties have # before their name. Maybe something similar is needed for static methods too?
Or just put (static)
I wonder how the original JSdoc template distinguishes it.
